ci-operator/step-registry/opendatahub-io/ci/s390x-build/dashboard/opendatahub-io-ci-s390x-build-dashboard-commands.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+set -ex
+
+# Session variables
+zvsi_fip=$(cat "${SHARED_DIR}/zvsi_fip")
+ssh_key_string=$(cat "${AGENT_IBMZ_CREDENTIALS}/httpd-vsi-key")
+export ssh_key_string
+tmp_ssh_key="/tmp/httpd-vsi-key"
+envsubst <<"EOF" >${tmp_ssh_key}
+-----BEGIN OPENSSH PRIVATE KEY-----
+${ssh_key_string}
+
+-----END OPENSSH PRIVATE KEY-----
+EOF
+chmod 0600 ${tmp_ssh_key}
+ssh_options=(-o 'PreferredAuthentications=publickey' -o 'StrictHostKeyChecking=no' -o 'UserKnownHostsFile=/dev/null' -o 'ServerAliveInterval=60' -i "$tmp_ssh_key")
+
+# Get current date
+current_date=$(date +%F)
+echo "Current date is $current_date"
+
+# Get IMAGE_TAG if not provided
+if [[ -z "$IMAGE_TAG" ]]; then
+    case "$JOB_TYPE" in
+        presubmit)
+            echo "Building default image tag for a $JOB_TYPE job"
+            IMAGE_TAG="pr-${PULL_NUMBER}"
+            if [[ -n "${RELEASE_VERSION-}" ]]; then
+                IMAGE_TAG="${RELEASE_VERSION}-${IMAGE_TAG}"
+            fi
+            ;;
+        postsubmit)
+            echo "Building default image tag for a $JOB_TYPE job"
+            IMAGE_TAG="${RELEASE_VERSION}-${PULL_BASE_SHA:0:7}"
+            ;;
+        periodic)
+            echo "Building default image tag for a $JOB_TYPE job"
+            IMAGE_TAG="${RELEASE_VERSION}-nightly-${current_date}"
+            IMAGE_FLOATING_TAG="${RELEASE_VERSION}-${YEAR_INDEX}"
+            ;;
+        *)
+            echo "ERROR Cannot publish an image from a $JOB_TYPE job"
+            exit 1
+            ;;
+    esac
+fi
+
+# Get IMAGE_TAG if it's equal to YearIndex in YYYYMMDD format
+if [[ "$IMAGE_TAG" == "YearIndex" ]]; then
+    YEAR_INDEX=$(echo "$(date +%Y%m%d)")
+    case "$JOB_TYPE" in
+        presubmit)
+            echo "Building YearIndex image tag for a $JOB_TYPE job"
+            IMAGE_TAG="pr-${PULL_NUMBER}"
```

Comment:
Wait, does Openshift CI offers `PULL_NUMBER` if it's triggered by a cron job? Doesn't it exclusive for prow triggers from a PR?

Reply:
As per the script , as we are providing RELEASE_VERSION for nightly builds, PULL_NUMBER  will be used for presubmit jobs , kept it as it is for future use.